platforms/core-runtime/launcher/src/integTest/groovy/org/gradle/launcher/daemon/DaemonInitialCommunicationFailureIntegrationSpec.groovy

```diff
@@ -198,18 +202,12 @@ class DaemonInitialCommunicationFailureIntegrationSpec extends DaemonIntegration
                 @Override
                 void run() {
                     while (true) {
-                        SocketChannel connection
                         try {
-                            connection = socket.accept()
+                            SocketChannel connection = socket.accept()
+                            connection.write(ByteBuffer.wrap("hello".getBytes(StandardCharsets.UTF_8)))
```

Comment:
@oehme Could you please add a comment here explaining why we don't close it yet?

Reply:
Done